wandb/sdk/internal/handler.py

```diff
@@ -826,7 +826,11 @@ def handle_request_sampled_history(self, record: Record) -> None:
             item.key = key
             values: Iterable[Any] = sampled.get()
             if all(isinstance(i, numbers.Integral) for i in values):
-                item.values_int.extend(values)
+                try:
+                    item.values_int.extend(values)
+                except ValueError:
+                    # it is safe to ignore these as this is for display information
```

Comment:
should we just add them float instead?

Reply:
that is definitely a valid option and consistent with how we are (mis)handling bigints in other places -- but id rather omit the data rather than mishandle them now.   When we have better handling of bigints, then we should figure out what the proper way to do this would be.